packages/clerk-js/src/core/resources/Base.ts

```diff
@@ -31,7 +32,10 @@ export abstract class BaseResource {
       clerkMissingFapiClientInResources();
     }
 
-    const { payload, status, statusText } = await BaseResource.fapiClient.request<J>(requestInit);
+    const { payload, status, statusText } = await runWithExponentialBackOff(
```

Comment:
⛔ This should be added initially to GET requests only. Adding it to mutating endpoints can result in unexpected behavior.

Reply:
Indeed! I thought it would be safe for all requests because this is for network errors, but maybe it succeeds and never reaches the client. Thank you!